src/SiteExtensions/build.cmd

```diff
@@ -10,8 +10,9 @@ IF %ERRORLEVEL% NEQ 0 (
 )
 
 ECHO Building LoggingBranch
-CALL %RepoRoot%\build.cmd -forceCoreMsbuild -arch x64 -projects %~dp0LoggingBranch\LB.csproj /bl:artifacts/log/SiteExtensions-LoggingBranch-x64.binlog %*
-CALL %RepoRoot%\build.cmd -forceCoreMsbuild -arch x86 -projects %~dp0LoggingBranch\LB.csproj /bl:artifacts/log/SiteExtensions-LoggingBranch-x86.binlog %*
+REM Remove /p:DisablePackageAssetsCache=true /p:DisableTransitiveFrameworkReferences=true which workarounds dotnet store and publish behaviour changes in preview 6
```

Comment:
This comment isn't clear. Is it a TODO for when we move to Preview 7 perhaps? Is there a bug tracking a fix in the SDK that should be linked here?

Reply:
I think we need a little more clarification on this matter, which I'll follow up with tomorrow. But to potentially unblock us, I wanted to test this change on the CI (works locally now). I'll send an email with a few more details to @mkArtakMSFT once I get a build pass/fail.